modules/ob-defining-annotations.adoc

```diff
@@ -0,0 +1,39 @@
+// This module is included in the following assembly:
+//
+// * builds/configuring-build-strategies.adoc
+
+:_content-type: REFERENCE
+[id="ob-defining-annotations_{context}"]
+= Annotations definition
+
+You can define annotations for a build strategy or a cluster build strategy like for any other Kubernetes object. Annotations are first propagated to the `TaskRun` resource. Then, Tekton propagates them to the pod. 
+
+You can use annotations for the following use cases:
+
+* To limit the network bandwidth the pod is allowed to use, the `kubernetes.io/ingress-bandwidth` and `kubernetes.io/egress-bandwidth` annotations are defined in the Kubernetes network traffic shaping feature.
+* To define the AppArmor profile of a container, the `container.apparmor.security.beta.kubernetes.io/<container_name>` annotation is used.
```

Comment:
Unsure about the construction here. To me, the UL doesn't clearly read as use cases--just facts about annotations.